user.go

```diff
@@ -165,25 +167,33 @@ func userfromCookie(cookie string, ip string) (u *User, err error) {
 		return nil, err
 	}
 
-	username, err := userFromAPI(claims.UserId)
+	username, createdAt, err := userFromAPI(claims.UserID)
 	if err != nil {
 		return nil, err
 	}
 
 	// if user not found, insert new user into db
 
-	// ignoring the error for now
-	db.newUser(claims.UserId, username, ip)
 	// TODO err is expected for non-new users...
+	// ignoring the error for now
+	_ = db.newUser(claims.UserID, username, ip)
 
 	// now get features from db, update stuff - TODO
 
-	features, uid, err := db.getUserInfo(claims.UserId)
+	now := time.Now()
+	t := time.Unix(createdAt, 0)
+
+	features, uid, err := db.getUserInfo(claims.UserID)
 	if err != nil {
 		fmt.Println("err4", err)
 		return nil, err
 	}
 
+	//	if now.Day() == t.Day() && now.Month() == t.Month() && now.Year() == t.Year() {
```

Comment:
the simplified user object needs to be regenerated when this changes. rn it will only work if the user logs in on their birthday and the flair will stay until they log in again